arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -594,6 +594,42 @@ def test_plot_pair_shapes(marginals, max_subplots):
     assert ax.shape == (side, side)
 
 
+@pytest.mark.parametrize("sharex", ["col", None])
```

Comment:
I'd prefer if testing was with backend_kwargs having a sharex key or not having the key. The fact that the key with a value of None is equivalent to no key is an implementation detail that users should not use. We could parametrize with True/False and then do something like

```
backend_kwargs = {}
if sharex:
    backend_kwargs["sharex"] = ...
...
```

same would happen also with sharey.

Reply:
That makes perfect sense (and is how I implemented it the first time, I can't remember why I changed it)... I'll revert it to mirror your suggestion now